.github/workflows/develop-publish-gh-pages.yml

```diff
@@ -0,0 +1,27 @@
+name: Deploy Action WebRTC JS SDK
+
+on:
+    push:
+        branches:
+            - develop
+jobs:
+    publish-ubuntu-build:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout repository
+              uses: actions/checkout@master
```

Comment:
Do we have to change the branch here to pull develop and not master?

Reply:
https://github.com/actions/checkout?tab=readme-ov-file#checkout-a-different-branch

Reply:
The `@master` is for the actual checkout action - it will use the latest master branch of the `checkout` action.

The action will check out the `develop` branch of the repository since it's specified in the `on` section.